src/content/en/2021/capabilities.md

```diff
@@ -1,20 +1,368 @@
+content="831",
+classes="big-number",
+sheets_gid="2077755325",
+sql_file="fugu.sql"
+)
+}}
+
+The capability is supported by Chromium browsers starting from version 67. The relative orientation sensor is used by 824 desktop and 831 mobile sites, the linear acceleration sensor by 257 desktop and 237 mobile sites, and the gyroscope by 36 desktop and 22 mobile sites. An example application that uses all three of them is <a hreflang="en" href="https://obs.ninja/">VDO.Ninja</a>, the former OBS Ninja. This software allows you to remotely connect with video broadcasting software such as OBS. The app allows the connected broadcasting software to read sensor data from the device. For example, to capture a smartphone's movements when streaming virtual reality content.
+
+## Sites using the most capabilities
+The analysis also identified the websites using the most capabilities from the HTTP Archive data set. The detection script is capable of identifying 30 Fugu APIs in total.
+
+1. The number one site is <a hreflang="en" href="https://whatwebcando.today/">whatwebcando.today</a>, which uses 28 capabilities. It showcases different HTML5 device integration APIs by providing a live demo for every capability. Naturally, the number of used APIs is very high. In the result set, a similar site called <a hreflang="en" href="https://whatpwacando.today/">whatpwacando.today</a> showcases PWA capabilities and uses eight APIs.
+1. The <a hreflang="en" href="https://polisnotis.se/">PolisNotis</a> PWA shows police notices in Sweden. It uses ten APIs, including the Declarative Link Capturing API to define that the PWA should always open a new window when clicking a PWA-related link. The Web Share API is used in the source code, but the sharing functionality is not exposed to the UI. The app also uses the Badging API to alert the user via the app icon if there is a new notice.
+1. The website <a hreflang="en" href="https://system-scanner.net/">System Scanner</a> uses nine APIs: It shows an overview of the system information exposed by the browser, including sensor information provided by the Generic Sensor API.
+1. Eight sites use eight Fugu APIs: One of them is the aforementioned <a hreflang="en" href="https://excalidraw.com/">Excalidraw</a>, an online drawing tool for creating drawings in a hand-drawn style. As a traditional productivity app, it benefits from the new capabilities.
+
+Some websites from the result set are Internet forums based on <a hreflang="en" href="https://www.discourse.org/">Discourse</a>. This forum software supports a total of eight Fugu APIs. Discourse-based forums are installable and support, among others, the Badging API to show the amount of unread notifications.
+
+The results also include sites that aren't proactively using the APIs. For example, some sites ship library code that could theoretically access the capabilities. Some sites check for the presence of Fugu APIs to determine the user's browser.
```

Comment:
I don't understand that last sentence? Do you mean they are using these APIs to fingerprint the user? Is that a risk of these APIs? Should we discuss that? Should we link to the Privacy chapter?

Also how does this work with the aforementioned need of interaction to use these APIs? Or is it allowed to test if the API exists (`if genericAPI in window`) without needing that interaction?

On a similar note to any of these need permission prompts before using? I would hope so! Should that be mentioned somewhere?

Reply:
> I don't understand that last sentence? Do you mean they are using these APIs to fingerprint the user? Is that a risk of these APIs? Should we discuss that? Should we link to the Privacy chapter?

Some of the sites I've seen test for the presence of certain APIs to determine the browser brand without looking at the user agent. I don't think this is considered fingerprinting. All specification texts contain a section about security and privacy considerations, we could mention that in the introduction. And yes, we can link to the privacy chapter, looks like they have a section on permissions.

> Also how does this work with the aforementioned need of interaction to use these APIs? Or is it allowed to test if the API exists (if genericAPI in window) without needing that interaction?

All APIs presented in this chapter can be detected without user interaction.

> On a similar note do any of these need permission prompts before using? I would hope so! Should that be mentioned somewhere?

Most APIs have implicit permission prompts (e.g., file picker, device picker, …), and it's always noted in the section about the API. Access to motion and light sensor APIs is granted by default.

Reply:
OK let's leave as is. Re-reading it it's fine. Resolving.